integration/query_array_test.go

```diff
@@ -47,18 +50,30 @@ func TestQueryArrayDotNotation(t *testing.T) {
 				Name:    "BadValue",
 				Message: "Can't have RegEx as arg to predicate over field 'v.array.0'.",
 			},
+			altMessage: "Can't have RegEx as arg to predicate over field 'v.array.0'.",
```

Comment:
This `altMessage` is identical to the error message.

```suggestion
```

Reply:
Also for the same reason https://github.com/FerretDB/FerretDB/pull/2806/files#r1222350133